src/components/button/index.js

```diff
@@ -1 +1,2 @@
 export * from './Button';
+export * from "./PillButton";
```

Comment:
Make sure to make an index.js file in the pillbutton folder and move this line there